src/main/java/com/google/ehub/data/ProfileDatastore.java

```diff
@@ -42,14 +42,13 @@ public  UserProfile createUserProfileFromEntity(Entity userEntity) {
 
     return new UserProfile(name, username, bio, email);
   }
-  public String getEmailProperty(){
+  public String getEmailProperty() {
     return EMAIL_PROPERTY_KEY;
   }
-  public String getItemKind(){
+  public String getItemKind() {
     return PROFILE_ITEM_KIND;
   }
-  public  DatastoreService getDatastore(){
-      return datastore;
+  public DatastoreService getDatastore() {
+    return datastore;
   }
```

Comment:
I don't think there is a need to provide access to the datastore service object through a method. Since anyone can just get it from the factory through the static method ```DatastoreServiceFactory.getDatastoreService()```. So I think it would just be better to remove the ```getDatastore()``` method from the class.